tests/Microsoft.Identity.Test.Integration.net47/Microsoft.Identity.Test.Integration.net47.csproj

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Instead of creating a new project, please update the existing one (Integration.net45) to target .NET 4.7.2.
It already targets .NET 4.6.2, so we're not missing anything.